pip-install: add --find-links example (#6319)
tldr-pages/tldr: pages/common/pip-install.md

`pip install -r {{requirements.txt}}`

> - Install packages from an URL or local file archive (.tar.gz | .whl):

Reviewer: ```suggestion
- Install packages from an URL or local file package:
```

Author: In my opinion archive is more precise than package here.

> - Install packages from an URL or local file archive (.tar.gz | .whl):

Author: ```suggestion
- Install packages from an URL or local file archive:
```

> `pip install -f {{url | filename}}`

Reviewer: ```suggestion
`pip install -f {{url|path/to/package_file.tar.gz|whl}}`
```

Reviewer 2: What do you mean by `whl` here?

Reviewer 2: I see. I'm not sure there's really a way to specify that easily. I'd pick the most common filetype and stick with it.

Reviewer: I don't know which is more common, however whl is the standard for python packages since its what its designed to do, while tar.gz is just a generic archive  

whl is specific, tar.gz is generic

I'd go with whl

Reviewer 3: Maybe we should just leave it out, since both options are listed in the description.

Reviewer 4: Maybe this is better
```suggestion
`pip install -f {{url|path/to/file}}`
```
